jupyterhub/jupyterhub-idle-culler/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Since this is kind of an "addon" to JH, it would make sense to me to make it into an overlay - WDYT?